### PR TITLE
[BER-FT] Sebastian - Elsa

### PR DESCRIPTION
**Pull request by ElsaFil**

### CODE REVIEW

File: starter-code/src/viking.js [hunk 1]
```diff
@@ -1,11 +1,100 @@
...
+  constructor(name, health, strength) {
+    super(health, strength);
+    this.name = name;
+  }
+
+  receiveDamage(damage) {
+    this.health -= damage;
+    if (this.health > 0) {
+      return this.name + " has received " + damage + " points of damage";
+    } else {
```
Comment (DalinaLeila) [inline starter-code/src/viking.js]: no need for the else here

File: starter-code/src/viking.js [hunk 2]
```diff
@@ -1,11 +1,100 @@
...
+      return this.name + " has died in act of combat";
```
Comment (DalinaLeila) [inline starter-code/src/viking.js]: Works fine, you might want to use backticks and the $ syntax. Will make the code even cleaner :)

File: starter-code/src/viking.js [hunk 3]
```diff
@@ -1,11 +1,100 @@
...
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier {
+  receiveDamage(damage) {
+    this.health -= damage;
+
+    if (this.health > 0) {
+      return "A Saxon has received " + damage + " points of damage";
+    } else {
```
Comment (DalinaLeila) [inline starter-code/src/viking.js]: see the comment above

**Comment by ElsaFil** [comment]
> Looks very good! 👍 👍 Attempt the super Bonus :)

Thanks for the comments! You can see the new changes and also the super bonus 😊💪

**Pull request closed without merging**